gen/build_deploy/powershell/install_choco.ps1

```diff
@@ -0,0 +1,204 @@
+# use an internal repository to download Chocolatey nupkg to the local path:
+# $packageRepo = '<INSERT REPO URL>'
+# uncomment if you want to download the file from a remote OData (HTTP/HTTPS) internal url (such as Artifactory, Nexus, ProGet, or Chocolatey.Server)
+# $searchUrl = ($packageRepo.Trim('/'), 'Packages()?$filter=(Id%20eq%20%27chocolatey%27)%20and%20IsLatestVersion') -join '/'
+
+# UPDATE THIS PATH
+$localChocolateyPackageFilePath = 'c:\dcos\chocolatey_offline   \chocolatey.0.10.15.nupkg'
+# Determine unzipping method
+# 7zip is the most compatible, but you need an internally hosted 7za.exe.
+# Make sure the version matches for the arguments as well.
+# Built-in does not work with Server Core, but if you have PowerShell 5
+# it uses Expand-Archive instead of COM
+$unzipMethod = 'builtin'
+#$unzipMethod = '7zip'
+#$7zipUrl = 'https://chocolatey.org/7za.exe' (download this file, host internally, and update this to internal)
```

Comment:
what is the purpose of this lines? They can be removed, right?

Reply:
Correct. 
install_choco.ps1 is officially published and available by chocolatey.org, however we are using this for offline installation.